### ISSUE TITLE
Slack like reactions

### ISSUE THREAD

**Issue by dignifiedquire** [haadcode/orbit#135]
Slack has the ability to react to individual messages with emojis, which is a great feature to express reactions to specific messages. Github does this as well, but only with a limited set of emojis. It would be great for Orbit to have this as well.

![screen shot 2016-10-04 at 19 35 24](https://cloud.githubusercontent.com/assets/790842/19085199/c3f3ed4c-8a69-11e6-86ce-36ee63242b3d.png)
![screen shot 2016-10-04 at 19 35 31](https://cloud.githubusercontent.com/assets/790842/19085200/c3fa2cca-8a69-11e6-85f8-111b201a04db.png)

**Comment by haadcode** [comment]
Definitely, would love to have this in Orbit! If anyone wants to work on it, ping me and I can get you started.

**Comment by evalsocket** [comment]
@haadcode i am here. where to start ?

**Comment by aphelionz** [comment]
@evalsocket I love your enthusiasm. Would you be interested in joining us on gitter in OrbitDB/Lobby? That might be a great place to start and get you going.

**Comment by evalsocket** [comment]
Yarn Audit result 
![screenshot 2019-01-08 at 1 39 55 am](https://user-images.githubusercontent.com/10830562/50791206-2f310d80-12e7-11e9-81f9-6ac6d855e879.png)

**Comment by RichardLitt** [comment]
@evalsocket Cool! Is it possible to use it here? Want to try integrating it in a PR?

**Comment by aphelionz** [comment]
I'd like to see that too - we will have to merge with @latenssi's `orbit-chat` work at some point soon, but I dont think this will make that merge any more or less problematic, honestly. I'll be a bit of work either way.

**Comment by evalsocket** [comment]
@RichardLitt  it is possible but i am thinking about UI/UX for that feature. @aphelionz is @latenssi  working on that feature ? if yes then where ?

**Comment by aphelionz** [comment]
@evalsocket Please see https://github.com/orbitdb/orbit-chat
